ci-operator/step-registry/rosa/cluster/provision/rosa-cluster-provision-commands.sh

```diff
@@ -78,6 +79,17 @@ else
   fi
   subfix=$(openssl rand -hex 2)
   CLUSTER_NAME=${CLUSTER_NAME:-"$prefix-$subfix"}
+  # For long cluster name enabled, append a "long_name_prefix_len" chars long random string to cluster name
+  # Max possible prefix length is 9, max possible subfix length is 4, hyppen is used 2 times, random string of length "long_name_prefix_len"
+  # (9 + 4 + 2 + "long_name_prefix_len" = 54 )
+  if [[ "$LONG_CLUSTER_NAME_ENABLED" == "true" ]]; then
+    long_name_prefix_len=$(( 54 - $(echo -n "$CLUSTER_NAME" | wc -c) - 1 ))
```

Comment:
better to use a const such as MAX_CLUSTER_NAME_LENGTH and put it at the beginning of the file (such as in line34), instead of directly using 54, since this length maybe changed in future and a const is more readable.
```
MAX_CLUSTER_NAME_LENGTH=54
long_name_prefix_len=$(( $MAX_CLUSTER_NAME_LENGTH - $(echo -n "$CLUSTER_NAME" | wc -c) - 1 ))
```

And we can also use a MAX_DOMAIN_PREFIX_LENGTH=15 somewhere to make the code more flexible and readable